userAggregateSimple.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
We don't need this file in here anymore, right? We have a (hopefully working) `userAggregate.php` now.